src/js/form-util.js

```diff
@@ -127,9 +128,21 @@ export function prepareInputs (formInputs, reasonInputs, reasonFieldset, reasonA
       return
     }
 
-    console.log(getProfile(formInputs), reasons)
+    const profile = getProfile(formInputs)
 
-    const pdfBlob = await generatePdf(getProfile(formInputs), reasons, pdfBase)
+    console.log(profile, reasons)
+
+    ;[
```

Comment:
Je ne suis pas un expert de JavaScript donc je peux me tromper, mais est-ce que ce point-virgule n'est pas en trop ?

Reply:
Sans le point virgule, javascript crois que le `[` est l'accès à l'élément d'un tableau qui serait retourné par la ligne au dessus

par exemple
```
console.log('test')
[1].forEach(e => console.log)
```
est interprété comme 
```
console.log('test')[1].forEach(e => console.log)
```
et throw une erreur "cannot read property '1' of undefined"

Reply:
Ce ne serait pas plus logique de mettre les trailing semicolons... à la fin de la ligne ? Ou ESLint l'empêche ?

Reply:
Y a tout un débat à ce sujet de puis des années https://www.google.com/search?q=javascript+semicolon+or+not voir également https://prettier.io/docs/en/rationale.html#semicolons

Reply:
https://dev.to/adriennemiller/semicolons-in-javascript-to-use-or-not-to-use-2nli

TLDR: en gros, au moment de compiler le code, JavaScript détermine les endroits où _il pense_ qu'il aurait dû y avoir des points-virgule. Sauf qu'il y a plein de cas où il se foire. Dans ces conditions, il me semble plus judicieux de mettre les points-virgule partout pour éviter les erreurs.
Bonus : ça rend le code plus accessible à ceux qui sont moins habitués aux subtilité du langage, et ça réduit un peu la complexité cognitive du code :)

Cela dit, il me semble plus sage de faire dans le cadre d'une autre issue pour éviter du bruit inutile sur cette PR (et ça sera plus propice aux discussions) :)